CMakeLists.txt

```diff
@@ -1,5 +1,9 @@
 cmake_minimum_required(VERSION 3.18 FATAL_ERROR)
 
+if(POLICY CMP0135)
+    cmake_policy(SET CMP0135 NEW)
+endif()
```

Comment:
What's this doing?

Reply:
Apparently it sets the extraction time to the time of extraction, rather than what is specified in the archive. I'm guessing this is because of caching or something.

Reply:
TL;DR - CMake 3.24 makes sure that downloaded items get rebuilt if the URL changes based on timestamp - doesn't affect lower versions as the policy doesn't exist, and w/o setting this the configuration fails in CI.